package.json

```diff
@@ -45,7 +45,6 @@
 		"algoliasearch": "^3.34.0",
 		"apollo-cache-inmemory": "^1.6.5",
 		"apollo-client": "2.6.8",
-		"apollo-fetch": "^0.7.0",
```

Comment:
were you previously using this for the Contentful stuff?

Reply:
Yeah, I've been testing this PR for a couple of days, while working on other things and that snuck in there, removed from this PR now